aiogram/webhook/aiohttp_server.py

```diff
@@ -184,6 +190,8 @@ async def _handle_request(self, bot: Bot, request: web.Request) -> web.Response:
         return web.Response(body=self._build_response_writer(bot=bot, result=result))
 
     async def handle(self, request: web.Request) -> web.Response:
+        if not self.secret_verify(request.headers.get("X-Telegram-Bot-Api-Secret-Token")):
```

Comment:
I believe this check should be performed after the bot instance is already known

This is required to check the secret for a specific bot instance